app/controllers/application_controller/miq_request_methods.rb

```diff
@@ -168,7 +168,7 @@ def set_pre_prov_vars
 
     unless %w[image_miq_request_new miq_template_miq_request_new].include?(params[:pressed])
       path_to_report = ManageIQ::UI::Classic::Engine.root.join("product", "views", provisioning_report).to_s
-      @view = MiqReport.new(YAML.safe_load(File.open(path_to_report), [Symbol]))
+      @view = MiqReport.new(YAML.load(File.open(path_to_report), [Symbol]))
```

Comment:
I'm not sure `YAML.load` accepts the whitelist...

Reply:
It looks like it silently works, but yes, it's incorrect to specify the whitelist with `load` and `safe_load` would need to enumerate all of the different types that  each report file could load.

```ruby
irb(main):001:0> YAML.load("--- !ruby/object:PidFile {}\n", [Symbol])
=> #<PidFile:0x00007fc607ab6920>
```